modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,64 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6139[OpenShift Logging Bug Fix Release 5.8.0] and link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0 Kibana].
+
+[id="logging-release-notes-5-8-0-deprecation-notice"]
+== Deprecation notice
+In Logging 5.8, Elasticsearch, Fluentd, and Kibana are deprecated and are planned to be removed in Logging 6.0, which is expected to be shipped alongside {product-title} version 4.16. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements. The Vector-based collector provided by the Cluster Logging Operator, and LokiStack provided by the Loki Operator are the preferred operators for log collection and storage.  We encourage all users to adopt the Vector and Loki log stack, as this will be the stack that will be enhanced going forward.
```

Comment:
I think it's best to avoid mentions of a specific OCP release that hasn't been released yet, and to be general if you can:
```suggestion
In Logging 5.8, Elasticsearch, Fluentd, and Kibana are deprecated and are planned to be removed in Logging 6.0, which is expected to be shipped alongside a future release of {product-title}. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements. The Vector-based collector provided by the Cluster Logging Operator, and LokiStack provided by the Loki Operator are the preferred operators for log collection and storage.  We encourage all users to adopt the Vector and Loki log stack, as this will be the stack that will be enhanced going forward.
```

In my suggestion, I kept the future logging version # since it's probably worth giving a notice that specific, but I felt that mentioning OCP 4.16 would tie us down a little too much. 

[Here's the RHSSG rule](https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future:~:text=Do%20not%20refer%20to%20a%20specific%20future%20release.%20For%20example%2C%20do%20not%20mention%20a%20particular%20release%20number%20or%20a%20specific%20release%20date.) about how we shouldn't mention future versions. Right below that rule is a note saying that deprecation notices are an exception, although I think the "have to specify a future release" refers to something like a legal requirement. So if there's no legal requirement making the mention of OCP 4.16 here necessary, or proof that this will ship alongside 4.16 with 100% certainty, I'd just leave it as "a future release".

**If you wish to keep the version references as is, please provide a PM ack somewhere clearly visible on this PR.** Thanks!

Reply:
@libander Should we specify the release number? Or mark it as a future release as @skopacz1 and @abrennan89 suggested.

Reply:
Discussed the change with Jamie. I'm keeping the change. Thanks! :)